apps/web/src/components/layout/components/v2/HeaderNav.tsx

```diff
@@ -19,18 +25,57 @@ import { WorkflowHeaderBackButton } from './WorkflowHeaderBackButton';
 import { SupportModal } from '../SupportModal';
 
 export function HeaderNav() {
-  const { currentUser } = useAuth();
+  const { currentUser, currentOrganization } = useAuth();
   const [isSupportModalOpened, setIsSupportModalOpened] = useState(false);
   const isV2Enabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_V2_ENABLED);
 
-  useBootIntercom();
+  // variable to check if it's the first render for. Needed for Plain live chat initialization
+  const [isFirstRender, setIsFirstRender] = useState(true);
+  const isLiveChatVisible =
+    process.env.REACT_APP_PLAIN_SUPPORT_CHAT_APP_ID &&
+    IS_NOVU_PROD_STAGING &&
+    currentOrganization?.apiServiceLevel !== 'free' &&
+    currentUser?.servicesHashes?.plain;
 
   const { Icon, themeLabel, toggleColorScheme } = useThemeChange();
 
   const toggleSupportModalShow = () => {
     setIsSupportModalOpened((previous) => !previous);
   };
 
+  useEffect(() => {
+    if (isLiveChatVisible && isFirstRender) {
+      // @ts-ignore
+      window.Plain.init({
+        appId: process.env.REACT_APP_PLAIN_SUPPORT_CHAT_APP_ID,
+        hideLauncher: true,
+        title: 'Chat with us',
+        links: [
+          {
+            icon: 'call',
+            text: 'Schedule a call with us',
```

Comment:
```suggestion
            text: 'Contact Sales',
```

This might be mis-understood as a support call